mftf/2.3/metadata.md

```diff
@@ -441,43 +459,94 @@ Sets one of the following operation types:
 Representation of a complex entity that may contain fields, arrays, and objects.
 An object must match the [entity] of the same `type`.
 
-Attribute|Type|Use|Description
----|---|---|---
-`key`|string|optional| Name of the object.
-`dataType`|string|required| Type of the related [entity].
-`required`|boolean|optional| Determines if the object is required or not. It must match the Magento REST API specification.
+| Attribute  | Type    | Use      | Description                                                                                    |
+| ---------- | ------- | -------- | ---------------------------------------------------------------------------------------------- |
+| `key`      | string  | optional | Name of the object.                                                                            |
+| `dataType` | string  | required | Type of the related [entity].                                                                  |
+| `required` | boolean | optional | Determines if the object is required or not. It must match the Magento REST API specification. |
 
 ### field {#field-tag}
 
-Representation of HTML form or REST API fields. 
+Representation of HTML form or REST API fields.
 
-Attribute|Type|Use|Description
----|---|---|---
-`key`|string|required|Name of the field. It must match the field name of the related [entity].
-`type`|string|optional|Type of the value. It may contain a primitive type or the type of another operation.
-`required`|boolean|optional| Determines if the field is required or not. It must match the Magento REST API specification.
+| Attribute  | Type    | Use      | Description                                                                                   |
+| ---------- | ------- | -------- | --------------------------------------------------------------------------------------------- |
+| `key`      | string  | required | Name of the field. It must match the field name of the related [entity].                      |
+| `type`     | string  | optional | Type of the value. It may contain a primitive type or the type of another operation.          |
+| `required` | boolean | optional | Determines if the field is required or not. It must match the Magento REST API specification. |
 
 ### array {#array-tag}
 
 Representation of an array.
 
-Attribute|Type|Use|Description
----|---|---|---
-`key`|string|required|Name of the array.
+| Attribute | Type   | Use      | Description        |
+| --------- | ------ | -------- | ------------------ |
+| `key`     | string | required | Name of the array. |
 
 It contains one or more `value` elements.
 
 ### value {#value-tag}
 
-An item in `array`.
+Declares a data type for items within `<array>`.
+
+1. Example of an array with value of a primitive data type:
```

Comment:
General question, should these be in a numbered list? Is there a specific task they are completing with the steps? Or you can remove the numbering and keep the bullets under each example.